authorization/token/repository/rpt_token.go

```diff
@@ -0,0 +1,189 @@
+package repository
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/fabric8-services/fabric8-auth/errors"
+	"github.com/fabric8-services/fabric8-auth/gormsupport"
+	"github.com/fabric8-services/fabric8-auth/log"
+
+	"github.com/goadesign/goa"
+	"github.com/jinzhu/gorm"
+	errs "github.com/pkg/errors"
+	"github.com/satori/go.uuid"
+)
+
+type RPTToken struct {
+	// This is the primary key value
+	TokenID uuid.UUID `sql:"type:uuid default uuid_generate_v4()" gorm:"primary_key;column:token_id"`
```

Comment:
This would be value of the `jti` claim in the token. ( JTI stands of JWT Token ID )

Reply:
Good to know, do you think `TokenID` is appropriate as an abstraction though?